docs/DeveloperGuide.md

```diff
@@ -282,22 +282,23 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `TBM` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: Adding a Client**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User meets secures a new business client.
+2.  User checks TBM to see if the client is somebody he knows previously.
```

Comment:
If the user meets a new business client then it would be someone that he doesn't know previously? This statement might be redundant if that is the case.

Reply:
Change to a clearer def. Since TBM performs basic contact management/tracking, the use case should be able to handle the scenario when the user inadvertently tries to add a duplicate entry for someone he might already have an entry. (Could be due to previous professional deals, working relations... etc that he forgot.). Duplicate entries would cause the archives to get messy, and so I added the case where TBM prevents/User does not add the duplicate entry.